sections/main-order.liquid

```diff
@@ -21,7 +21,7 @@
     </symbol>
   </svg>
   <div>
-    <h1>{{ 'customer.account.title' | t }}</h1>
+    <h1 class="customer__title">{{ 'customer.account.title' | t }}</h1>
```

Comment:
just added this class here with Tyler's approval to fix the `margin-top` issue I mentioned [here](https://github.com/Shopify/dawn/pull/1640#discussion_r875999212)